README.md

```diff
@@ -183,9 +183,48 @@ Terraform will perform the following actions:
         ]
 ```
 
+## Importing an existing subscription
+
+Note before you begin, one of the resources to be imported is a service endpoint for Azure DevOps. This will require you to create a Personal Access Token with `Service Connections (Read, query & manage)` permissions which you can use in an environment variable along with the organisation URL to authenticate the microsoft/azuredevops terraform provider. See https://registry.terraform.io/providers/microsoft/azuredevops/latest/docs#argument-reference. 
```

Comment:
I think this will look better, and switched to sentence per line to make review easier.
```suggestion
_Note: before you begin, one of the resources to be imported is a service endpoint for Azure DevOps.
This will require you to create a Personal Access Token with `Service Connections (Read, query & manage)` permissions which you can use in an environment variable along with the organisation URL to authenticate the microsoft/azuredevops terraform provider.
See [the provider documentation](https://registry.terraform.io/providers/microsoft/azuredevops/latest/docs#argument-reference)._
```